### DISCUSSION TITLE
Whitelist good apps

### DISCUSSION THREAD

**Discussion by unalignedcoder** [FunkyMuse/Vigilante#593] [Ideas]
It seems counterintuitive that I am being warned of the camera being used when I open the camera app, microphone being used when I use the sound recorder, and so forth.

There should be a way to whitelist apps that are being deliberately employed by the user, as well as whitelisting manual settings such as turning on location from quick tiles. 

BTW, I am pretty sure my wife's Instagram listens to our conversations to plug products, but would this app catch it?

**Comment by FunkyMuse** [comment]
feel free to open a PR

**Reply by unalignedcoder** [reply]
First I need to find out what a PR is

**Reply by FunkyMuse** [reply]
https://www.pagerduty.com/resources/learn/what-is-a-pull-request/
